### PR TITLE
counter-strike cheat sheet

### PR DESCRIPTION
**Pull request by santoshdistro**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [ ] New Instant Answer
  - [x] Cheat Sheets: **`New {counter-strike} Cheat Sheet`**
###### Description of changes

show all the button of keyboard use to play the game

### CODE REVIEW

**Comment by daxtheduck** [comment]
@santoshdistro Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    "section_order":[
+        "Movement",
+        "Communication",
+        "Menu",
+        "Combat",
+        "Miscellaneous",
+        "Advanced"
+    ],
+    "sections":{
+        " Movement":[
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Consider removing space before `Movement`

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 2]
```diff
@@ -0,0 +1,263 @@
+{
+    "id":"Counter_Strike_cheat_sheet",
+    "name":"Counter Strike",
+    "description": "Counter Strike is well known game played widely globally",
+    "metadata":{
+        "sourceName":"Counter Strike",
+        "sourceUrl":"https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly"
+    },
+  "aliases": [
+    "counter strike"
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: The alias name can't be same as the cheat sheet name.Lets remove this.

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 3]
```diff
@@ -0,0 +1,263 @@
+{
+    "id":"Counter_Strike_cheat_sheet",
+    "name":"Counter Strike",
+    "description": "Counter Strike is well known game played widely globally",
+    "metadata":{
+        "sourceName":"Counter Strike",
+        "sourceUrl":"https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly"
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Are you sure that this sourceUrl should be here. :wink:  Please consider replacing this. :smile:

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+        "Communication",
+        "Menu",
+        "Combat",
+        "Miscellaneous",
+        "Advanced"
+    ],
+    "sections":{
+        " Movement":[
+            {
+                "val":" Move forward",
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Consider removing space before `Move forward`

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+                "key":"[↑]"
+            },
+            {
+                "val":" Move backward",
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Consider removing space before `Move backward`

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+                "key":"[↓]"
+            },
+            {
+                "val":" Turn left",
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Consider removing space before `Turn left`

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+                "key":"[←]"
+            },
+            {
+                "val":" Move left (strafe)",
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Consider removing space before `Move left`

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+                "key":"[A]"
+            },
+            {
+                "val":" Move right (strafe)",
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Lets remove spaces for all of the entries :smile:

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 9]
```diff
@@ -0,0 +1,263 @@
...
+            {
+                "val":" Take screenshot",
+                "key":"[F5]"
+            },
+            {
+                "val":"Cheer",
+                "key":"[J]"
+            },
+            {
+                "val":"Quit game ",
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Lets remove space after `Quit game`

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**